helm/default-apps-aws/values.yaml

```diff
@@ -7,6 +7,12 @@ userConfig:
       values: |
         ipam:
           mode: kubernetes
+        k8sServiceHost: api.{{ .Values.baseDomain }}
+        k8sServicePort: 6443
+        kubeProxyReplacement: strict
+        hubble:
+          relay:
+            enabled: true
```

Comment:
Not a blocker, but why don't we enable this by default, in the app?
We should IMHO strive to make cilium configuration as equal as possible between providers, and having custom settings like this one in provider-specific repos me don't like

Reply:
I agree with that. Do you know if that could break something in vintage? Currently it's like that for CAPG and CAPA. If you say it's fine on vintage, I can move that config to the app itself and remove it from here.

Reply:
In WCs cilium is still a dream.
In MCs I will test it with next bunch of releases I guess.
Safe to change the default if you ask me. 
Maybe worth asking rocket as well